fees/jito-restaking/index.ts

```
@@ -0,0 +1,43 @@
+
+import { FetchOptions, SimpleAdapter } from "../../adapters/types"
+import { CHAIN } from "../../helpers/chains"
+import { getSqlFromFile, queryDuneSql } from "../../helpers/dune"
+
+const fetchRevenue = async (options: FetchOptions) => {
+  // Jito Restaking Related revenue
+
+  // https://dune.com/queries/4908531
+  const sql = getSqlFromFile("helpers/queries/jito-restaking.sql", {
+    start: options.startTimestamp,
+    end: options.endTimestamp
+  });
+
+  const data: any[] = (await queryDuneSql(options, sql));
+  if (data[0] == undefined) {
```

we prefer if it throws an error, instead of handling and returning zero value.